packages/keystone/lib/List/index.js

```diff
@@ -152,8 +152,8 @@ module.exports = class List {
     const nonIdFieldNames = Object.keys(fields).filter(k => k !== 'id');
```

Comment:
Just wondering if we should account for the case of fields with `read: false`, as these probably shouldn't be used as `defaultColumns` or `defaultSort`.

Reply:
On closer inspection we don't have that info available at this stage, so this is fine for now.